### ISSUE TITLE
Cannot sign into my account on chatbotui.com

### ISSUE THREAD

**Issue by taksyon** [mckaywrigley/chatbot-ui#1751]
I made an account when I set this app up on my system awhile ago, and I wanted to sign in to manage my account on the website. However, it says "invalid login credentials". I understand that the account was primarily to use the app while hosting locally, but is the account I created also not stored on the web server?

**Comment by tonykay** [comment]
Yes, I saw the same issue yesterday. While the UI offered a reset button, no email came—I tried several times.

**Comment by gewei20** [comment]
Have you solved this problem? I also have this problem
